meshroom/core/node.py

```diff
@@ -1386,7 +1395,7 @@ def has3DOutputAttribute(self):
     internalAttributes = Property(BaseObject, getInternalAttributes, constant=True)
     internalAttributesChanged = Signal()
     label = Property(str, getLabel, notify=internalAttributesChanged)
-    color = Property(str, getColor, notify=internalAttributesChanged)
+    color = Property(str, getColor, setColor, notify=internalAttributesChanged)
```

Comment:
(same as above) Adding the setter is not needed in this scope.

Reply:
This has been removed.